docs/user-guide/configuring.md

```diff
@@ -391,6 +391,21 @@ The complete configuration hierarchy, from highest precedence to lowest preceden
     1. `eslint.json`
     1. Blank (no config)
 
+## Extending Configuration Files
+
+If you want to extend an existing configuration file which is not located in a parent directory, you can use the `extends` field to specify the location. For example:
```

Comment:
This is really any file, including one in a parent directory.

We probably also need to mention that the extended config provides base rules that are overridden by this config.